src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -64,22 +65,29 @@
 	private final HashSet<ICellProvider> inactiveCellProviders = new HashSet<>();
 	private final SetMultimap<IAEStack, ItemWatcher> interests = HashMultimap.create();
 	private final GenericInterestManager<ItemWatcher> interestManager = new GenericInterestManager<>( this.interests );
-	private final NetworkMonitor<IAEItemStack> itemMonitor = new NetworkMonitor<>( this, StorageChannel.ITEMS );
-	private final NetworkMonitor<IAEFluidStack> fluidMonitor = new NetworkMonitor<>( this, StorageChannel.FLUIDS );
+	// private final NetworkMonitor<IAEItemStack> itemMonitor = new NetworkMonitor<>( this,
+	// AEApi.instance().storage().getStorageChannel( IItemStorageChannel.class ) );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)